src/docs/windows/pwa-api-documentation/pwa-api-documentation.md

```diff
@@ -0,0 +1,312 @@
+
+<img src="/docs/windows/pwa-api-documentation/in-store.png" width="350px" alt="What PWA installation from the Microsoft Store looks like" /> 
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    <a href="https://www.pwabuilder.com" target="_blank" rel="noopener">PWABuilder</a> can automatically package and submit Chromium Based Edge PWAs to the store, ensuring that your PWA works the same whether its installed from the Store or Edge itself.
+  </p>
+</div>
+
+## App Experience on Windows
+Once your PWA has been installed by the user (either through the Store or Edge) Windows will treat this PWA just like any other app on Windows. Your PWA can be pinned to the Taskbar and Start Menu:
+
+<img src="/docs/windows/pwa-api-documentation/taskbar.png" width="350px" alt="A screenshot of a PWA in the taskbar" /> 
+
+Your PWA will also open in its own Window:
+
+<img src="/docs/windows/pwa-api-documentation/in-window.png" width="350px" alt="A screenshot of the PWA running in its own window" /> 
+
+And finally, it will also show up in your App Settings:
+
+<img src="/docs/windows/pwa-api-documentation/app-settings.png" width="350px" alt="A screenshot of the PWA in the settings app" /> 
+
+## File Handling and Integrating with the File Explorer
+
+Handling files has always been a challenge for Web Apps as they have never had the same file handling capability as native apps. For example, you could not simply open a file from the File System, download a file or securely write to a file on the users device. With the [File System API](https://web.dev/file-system-access/), your PWA can now use files with the normal experience users expect from apps.
+
+Your PWA can prompt the user to open a file, open mutiple files, open a directory and save files! Your PWA can also prompt the user for write access to an existing file, allowing your pwa to transparently write to a file without any extra downloads needed.
+
+<div class="demo-block">
+  <strong>Demo</strong>
+
+  <p>
+    Check out a demo <a href="https://wiggly-impossible-crayfish.glitch.me/file-handling.html" target="_blank" rel="noopener">here</a> of the File System Access API!
+  </p>
+</div>
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://web.dev/file-system-access/" target="_blank" rel="noopener">this tutorial</a> to learn how to use the File System Access API and enable this functionality in your PWA.
+  </p>
+</div>
+
+## Sharing
+
+Sharing content, and ingesting shared content, has always required custom UI and logic from the developer, until today! Your PWA can easily share content using the native Share UI built into Windows:
```

Comment:
A visual might help here, where websites have a bunch of share UI e.g. at the bottom of an article. You could note how this also hurts web performance, as additional images, CSS, and JS (sometimes, even 3rd party scripts) need to be run in order to offer a sub-par share experience.